docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.4"
+
+x-logging:
+  &default-logging
+  driver: "json-file"
+  options:
+    max-size: "50m"
+    max-file: "10"
+
+services:
+  stac:
+    container_name: stac-populator-test
+    image: ghcr.io/crim-ca/stac-app:main
+    depends_on:
+      - stac-db
+    ports:
+      - "8880:8000"
+    environment:
+      - POSTGRES_USER=dchandan
```

Comment:
do you want to use your username here? you could just change it to something generic

Reply:
yeah will do. The docker stuff is just for testing, it will be removed later.